main.go

```diff
@@ -31,22 +32,23 @@ func main() {
 	flag.StringVar(&configFile, "config", "", "(Optional) Config file")
 	flag.StringVar(&configFile, "c", "", "(Optional) Config file")
 	flag.Parse()
-
 	viper.SetConfigName("cps")
 	viper.AddConfigPath("/etc/cps/")
 	viper.AddConfigPath(".")
-	viper.SetEnvPrefix("cps")
-	// Allow dev mode to be set via env var
-	viper.BindEnv("dev") //nolint: errcheck
-
 	if configFile != "" {
 		viper.SetConfigFile(configFile)
+		err := viper.ReadInConfig()
+		if err != nil {
+			panic(fmt.Sprintf("Fatal error reading in config file: %s", err))
+		}
```

Comment:
As per slack conversation, to catch all usecases i think this should become:

```suggestion
	if configFile != "" {
		viper.SetConfigFile(configFile)
	}
	err := viper.ReadInConfig()
	if err != nil {
		fmt.Sprintf("Error reading in config file: %s. Will proceed with CPS_* environment variables", err))
	}
```